src/stocastic_functions/cost_functions.jl

```diff
@@ -0,0 +1,21 @@
+"""
+Returns the amount of misclassified examples, i.e. how many tuples with non-matching entries are there in `results`.
+# Arguments
+- `results::AbstractVector{Tuple{Int64, Int64}}`: the vector of tuples, where each tuple is in the form `Tuple{expected_output, actual_output}`.
+"""
+function misclassification(results::AbstractVector{Tuple{Int64,Int64}})
+    return count(pair -> pair[1] != pair[2], results) / length(results)
+end
+
+"""
+Returns the mean squared error of `results`.
+# Arguments
+- `results::AbstractVector{Tuple{Int64, Int64}}`: the vector of tuples, where each tuple is in the form `Tuple{expected_output, actual_output}`.
+"""
+function mean_squared_error(results::AbstractVector{Tuple{Int64,Int64}})
```

Comment:
Should we change `Tuple{Int64,Int64}` with `Tuple{<:Number,<;Number}` to make it more general? I.e., you don't need a different MSE function if the base numeric type changes

Reply:
This is a very good point. The error could also be a floating point number.